eqcorrscan/utils/plotting.py

```diff
@@ -1196,7 +1195,8 @@ def noise_plot(signal, noise, normalise=False, **kwargs):
             continue
         n_traces += 1
 
-    fig, axes = plt.subplots(n_traces, 2, sharex=True)
+    fig, axes = plt.subplots(n_traces, 2, sharex=True,
+                             figsize=(n_traces, n_traces))
```

Comment:
I'm concerned that this would become massive for very large numbers of traces. Maybe set a maximum-size? e.g.:
```python
fig_dim = min(n_traces, 10)
... figsize=(fig_dim, fig_dim))
```